python/rikai/spark/functions/vision.py

```diff
@@ -453,3 +455,36 @@ def crop(img: Image, box: Union[Box2d, List[Box2d]]):
 
     """
     return img.crop(box)
+
+
+@udf(returnType=ImageType())
+def inspect_bbox(img: Image, boxes: List[Box2d], labels: List[str]):
+    """Draw bounding boxes on the image, and the returns the inpected image
```

Comment:
Do we need to put this into a SQL as fixed open API from Rikai?

For example, how should we define color, font, texture, transparency, location of texts to boxes? Or do we want to?

Reply:
I think this simple bounding boxes drawer udf is sufficient for Rikai.

UDF can be easily adapted by nature. I guess there are professional pip packages to draw bounding boxes.

Reply:
For example: https://github.com/varunagrawal/bbox

Personally, I'm in favor of keeping only the necessary dependencies. For advanced usage, we could put it in the contrib folder.